src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,12 @@
 import React from 'react';
+import { User } from '../../types/User';
+import './UserInfo.scss';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User;
+};
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user?.email}`}>
+    {user?.name}
+  </a>
```

Comment:
in this case, if you do not have a user, you still will have a tag an in your HTML (it will be with an empty text ad with a broken link).
And + if you will not have user - you will still have words:` Posted by:` and then nothing after
![image](https://user-images.githubusercontent.com/125828769/231584986-214b0bc5-5bd8-4fdd-ab3e-6f92b11dc1f4.png)